e2e/eslint/src/linter.test.ts

```diff
@@ -248,9 +248,9 @@ describe('Linter', () => {
       const libC = uniq('tslib-c');
 
       beforeAll(() => {
-        runCLI(`generate @nx/js:lib ${libA}`);
-        runCLI(`generate @nx/js:lib ${libB}`);
-        runCLI(`generate @nx/js:lib ${libC}`);
+        runCLI(`generate @nx/js:lib ${libA} --bundler=none`);
```

Comment:
Ensure dep-checks do not catch those libs, since this test is not supposed to be checking it.